saleor/giftcard/migrations/0019_auto_20230623_0857.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.2.18 on 2023-06-23 08:57
+
+import django.contrib.postgres.indexes
+import django.contrib.postgres.search
+from django.db import migrations, models
+from django.contrib.postgres.operations import AddIndexConcurrently
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("giftcard", "0018_metadata_index"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="giftcard",
+            name="search_index_dirty",
+            field=models.BooleanField(db_index=True, default=True),
+        ),
+        migrations.AddField(
+            model_name="giftcard",
+            name="search_vector",
+            field=django.contrib.postgres.search.SearchVectorField(
+                blank=True, null=True
+            ),
+        ),
+        AddIndexConcurrently(
```

Comment:
I believe there is something wrong here. AddIndexConcurrently needs `atomic=False`, but I can't see it. If the migrations is running correctly then probably we need to remove statement from line 18 `db_index=True`